tool/gravity/cli/status.go

```diff
@@ -301,11 +305,14 @@ func formatVersion(version *proto.Version) string {
 	return "n/a"
 }
 
-func printClusterStatus(cluster statusapi.Cluster, w io.Writer) {
+func printClusterStatus(cluster statusapi.Cluster, w io.Writer, site *ops.Site) {
 	if cluster.App.Name != "" {
 		fmt.Fprintf(w, "Cluster image:\t%v, version %v\n", cluster.App.Name,
 			cluster.App.Version)
 	}
+	if site.Provider != "" {
+		fmt.Fprintf(w, "Cloud provider:\t%v\n", site.Provider)
```

Comment:
Like mentioned above, this should already be available in `cluster` passed here?